include/clap/ext/draft/midici-profiles.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "../../plugin.h"
+
+//usage examples:
+//- the host can tell a MIDI keyboard the plugin conforms to the drawbar organ profile, so the keyboard can set up its faders for that.
+//- the plugin can adapt its MIDI controller mapping if a MIDI keyboard actives a profile (via the host).
+//- the host can send per-note articulations in note attributes if there's an active profile for this.
+
+//the 5 bytes repesenting a profile go in a uint64_t.

```

Comment:
I would make it a typedef instead of using uint64_t directly. This would be used by the getter and setter and close to the typedef declaration/definition there could be the explanation about the 5 bytes and the example.